### DISCUSSION TITLE
Chunk naming

### DISCUSSION THREAD

**Discussion by vuki656** [vercel/next.js#18981] [Help] [answered]
I have a question regarding how chunks are named.

Does the name of the chunk change for any other reason other than content (code) change?

I noticed if you build the project for the second time (after deleting the .next folder), the chunk names will remain the same for chunks with no content changes.

**Comment by jamesmosier** [comment, marked as answer]
Next uses webpack's [`contenthash`](https://webpack.js.org/guides/caching/) for naming the bundles/chunks, therefore this only changes when the content changes.

**Reply by saurishkar** [reply]
Hey @jamesmosier 
Facing an issue regarding this in Next 12.3.4. If I try to create successive builds, the `webpack` chunk id changes but all other chunk ids remain the same. If I downgrade it to Next 12.1.0, this issue is fixed. Any suggestions ?
